packages/markdown/src/index.ts

```diff
@@ -47,7 +47,7 @@ export const markdownStaticHandler = (
 
         const isDir = !(dir === '')
 
-        return /\.(md|markdown)/.test(ext) && unPrefixedURL === (isDir ? `${dir}/${name}` : name)
+        return /\.(md|markdown)/.test(ext) && unPrefixedURL === (isDir ? `${dir}/${name}`.replace('\\', '/') : name)
```

Comment:
why did you add this and how it's related to OpenAPI validation?

Reply:
It's not related to OpenAPI validation. I'm using Windows, and I can't commit unless all the tests pass due to some pre-commit hooks. Markdown uses recursiveReadDir or promise from readdirp, which will print paths differently based on the operating system. Windows uses backslashes in its directory path, while Unix uses slashes. For example, on Windows:
```
PS C:\Users\sisyphus\source\repos\tinyhttp\packages\markdown\src> node
Welcome to Node.js v15.2.0
Type ".help" for more information.
var recursiveReadDir = require('readdirp').promise;
undefined
> recursiveReadDir('.').then((value) => { console.log(value); })
Promise { <pending> }
> [
  {
    path: 'index.ts',
    fullPath: 'C:\\Users\\sisyphus\\source\\repos\\tinyhttp\\packages\\markdown\\src\\index.ts',
    basename: 'index.ts',
    dirent: Dirent { name: 'index.ts', [Symbol(type)]: 1 }
  }
]
```
If we look at the following segment:
https://github.com/talentlessguy/tinyhttp/blob/b78a2d60766b4d22c9e55053c09d1d95d942ce45/packages/markdown/src/index.ts#L45-L50

we see that unprefixedUrl is being compared with the expression ``(isDir ? `${dir}/${name}` : name)``, but unprefixedUrl is a parsed url (`/talentlessguy/tinyhttp/pull/188` for example), which always uses forward slashes. So if we use the URL of this github pull request for example, the code would be comparing `/talentlessguy/tinyhttp/pull/188` with `\\talentlessguy\\tinyhttp\\pull\\188`; on Windows this results in an error for one of the markdown tests.

Reply:
So the replace is needed to change backslashes into slashes so when testing on Windows, `unprefixedUrl` and `(isDir ? `${dir}/${name}` : name)` equal each other for a valid GET request accessing a markdown file that exists.

Reply:
ah, no problem then 

thanks for fixing the bug! :D